core/modules/offline-order/helpers/onNetworkStatusChange.ts

```diff
@@ -10,8 +10,7 @@ export function onNetworkStatusChange (store) {
 
   if (typeof navigator !== 'undefined' && navigator.onLine) {
     EventBus.$emit('sync/PROCESS_QUEUE', { config: config }) // process checkout queue
-    store.dispatch('cart/load')
-
+    store.dispatch('cart/load', { forceClientState: true })
```

Comment:
shouldn't be changed if default value is `true`
```suggestion
    store.dispatch('cart/load')
```